package.json

```diff
@@ -382,7 +382,7 @@
     "tar": "4.4.13",
     "tinycolor2": "1.4.1",
     "tinygradient": "0.4.3",
-    "topojson-client": "3.0.0",
+    "topojson-client": "3.1.0",
```

Comment:
Match the current topojson-client version used by ems-client. Should reduce the bundle size just a little bit.